### PR TITLE
Create personal-finances.md

### PR DESCRIPTION
**Pull request by dmurko**
Ref: https://github.com/niteoweb/operations/issues/1389

### CODE REVIEW

**Review by marbeann — APPROVED** [review]

**Review by karantan — APPROVED** [review]

File: 5_People/personal-finances.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+We're assuming that:
+
+- You're investing for the long term (decades, not years, much less months).
+- You don't need the money now or in a year (better yet: only after retirement). 
+- You have the discipline to keep to a strategy even when it looks like it's not working. This means not selling when there is a panic in a market.
+
+When it comes to investing in stocks (company shares) and bonds (loans with fixed interest from governments and corporations), data has shown nobody can beat the market average long-term (decades). Therefore, your goal is to replicate the market as much as possible and at the lowest cost possible. Stock picking, while fun, is closer to the lottery than investing. You might get lucky, but more probably, you'll have less at the end than if you invested in the market. Therefore, we'll be investing in funds that own billions worth of stocks and bonds.
+
+What is the average yearly yield for long-term investing? 7%. It does not sound much but compounded it doubles your money every 10 years.
```
Comment (zupo) [inline 5_People/personal-finances.md:68]: It's closer to 5% when you take the whole World and not just US history, which you should.

https://rationalreminder.ca/podcast/146: `the world index geometric return was 5.2% over this period and the US was 6.6%.`

**Review by nkk0 — APPROVED** [review]

File: 5_People/personal-finances.md (hunk 1 above)
Comment (zupo) [reply 5_People/personal-finances.md:68]: So then it's `1.05^14 = 1.98`, meaning it doubles every 14 years.

Comment (dmurko) [reply 5_People/personal-finances.md:68]: @zupo Is that after inflation?

Comment (dmurko) [reply 5_People/personal-finances.md]: Updated to 

> What is the average yearly yield for long-term investing? <a href="https://rationalreminder.ca/podcast/146" target="_blank">5.2%</a>. It does not sound much but compounded it doubles your money every 14 years and that's after inflation which would in the same time frame decrease your purchasing power by X%.

I'm researching X.

**Comment by dmurko** [comment]
@zupo 

> What is the average yearly yield for long-term investing? [5.2%](https://rationalreminder.ca/podcast/146) which compounded doubles your money every 14 years. That does not sound like a lot but it is *after* inflation. In the same timeframe, inflation would decrease your purchasing power by 29% (average 2007-2021).

**Review by suryasr007 — APPROVED** [review]

**Pull request merged**